multiplex test_proteinase_k_filter_deltacchalf[None] failing

`$ pytest --regression-full "Test/regression/test_multiplex.py::test_proteinase_k_filter_deltacchalf[None]"`
```
___ test_proteinase_k_filter_deltacchalf[None] ___

d_min = None, regression_test = None, dials_data = <DataFetcher: /dls/science/groups/scisoft/DIALS/dials_data>
tmpdir = local('/tmp/pytest-of-wra62962/pytest-2415/test_proteinase_k_filter_delta0')

    @pytest.mark.parametrize(
        "d_min", [None, 2.0],
    )
    def test_proteinase_k_filter_deltacchalf(d_min, regression_test, dials_data, tmpdir):
        data_dir = dials_data("multi_crystal_proteinase_k")
        expts = sorted(f.strpath for f in data_dir.listdir("experiments*.json"))
        refls = sorted(f.strpath for f in data_dir.listdir("reflections*.pickle"))
        with tmpdir.as_cwd():
            command_line_args = (
                expts
                + refls
                + [
                    "filtering.method=deltacchalf",
                    "filtering.deltacchalf.stdcutoff=1",
                    "max_clusters=1",
                    "nproc=1",
                    "resolution.d_min=%s" % d_min,
                ]
            )
            run_multiplex(command_line_args)
        for f in expected_data_files + [
            "filtered.expt",
            "filtered.refl",
            "filtered.mtz",
            "filtered_unmerged.mtz",
        ]:
            assert tmpdir.join(f).check(file=1), "expected file %s missing" % f
        assert len(load.experiment_list(tmpdir / "scaled.expt", check_format=False)) == 8
        assert len(load.experiment_list(tmpdir / "filtered.expt", check_format=False)) < 8
    
        # assert that the reflection files are different - the filtered reflections
        # should have fewer reflections as one data set has been removed
        mtz_scaled = iotbx.mtz.object(tmpdir.join("scaled_unmerged.mtz").strpath)
        mtz_filtered = iotbx.mtz.object(tmpdir.join("filtered_unmerged.mtz").strpath)
        if d_min:
            # assert that the input d_min has carried through to the output files
            for mtz in (mtz_scaled, mtz_filtered):
                assert mtz.as_miller_arrays()[0].d_min() == pytest.approx(d_min, abs=1e-4)
        else:
            # If we've got rid of a the worst data set then the estimated resolution should
            # have improved
>           assert (
                mtz_filtered.as_miller_arrays()[0].d_min()
                < mtz_scaled.as_miller_arrays()[0].d_min()
            )
E           assert 2.3600090653517376 < 2.3000557760017406
E            +  where 2.3600090653517376 = <bound method set.d_min of crystal.symmetry(\n    unit_cell=(68.49859619, 68.49859619, 103.9505005, 90, 90, 90),\n    space_group_symbol="P 41 21 2"\n  )>()
E            +    where <bound method set.d_min of crystal.symmetry(\n    unit_cell=(68.49859619, 68.49859619, 103.9505005, 90, 90, 90),\n    space_group_symbol="P 41 21 2"\n  )> = <[AttributeError("'array' object has no attribute 'typecode'",) raised in repr()] array object at 0x7fa4a4c3fd30>.d_min
E            +  and   2.3000557760017406 = <bound method set.d_min of crystal.symmetry(\n    unit_cell=(68.49859619, 68.49859619, 103.9505005, 90, 90, 90),\n    space_group_symbol="P 4 2 2"\n  )>()
E            +    where <bound method set.d_min of crystal.symmetry(\n    unit_cell=(68.49859619, 68.49859619, 103.9505005, 90, 90, 90),\n    space_group_symbol="P 4 2 2"\n  )> = <[AttributeError("'array' object has no attribute 'typecode'",) raised in repr()] array object at 0x7fa4a4d5c5f8>.d_min

Test/regression/test_multiplex.py:115: AssertionError
```

--- a/Test/regression/test_multiplex.py
+++ b/Test/regression/test_multiplex.py
@@ -109,13 +109,7 @@ def test_proteinase_k_filter_deltacchalf(d_min, regression_test, dials_data, tmp
         # assert that the input d_min has carried through to the output files
         for mtz in (mtz_scaled, mtz_filtered):
             assert mtz.as_miller_arrays()[0].d_min() == pytest.approx(d_min, abs=1e-4)
-    else:
-        # If we've got rid of a the worst data set then the estimated resolution should
-        # have improved
-        assert (
-            mtz_filtered.as_miller_arrays()[0].d_min()
-            < mtz_scaled.as_miller_arrays()[0].d_min()
-        )
+
     assert mtz_filtered.n_reflections() != mtz_scaled.n_reflections()
 
     with tmpdir.join("xia2.multiplex.json").open("r") as fh:

